docs/relier-communication-protocols/sync-webchannel.md

```diff
@@ -0,0 +1,133 @@
+SYNC via WebChannels protocol
```

Comment:
nit: `SYNC` is this only for sync of any communication between Client and Content server?